Changelog.md

```diff
@@ -2,6 +2,67 @@
 
 ### Ignition Gazebo 4.X.X (20XX-XX-XX)
 
+### Ignition Gazebo 4.1.0 (2020-12-11)
+
+1. Update Dockerfiles to use focal images
+    * [pull request 388](https://github.com/ignitionrobotics/ign-gazebo/pull/388)
+
+1. Updated source build instructions for ign-gazebo4
+    * [pull request 404](https://github.com/ignitionrobotics/ign-gazebo/pull/404)
+
+1. Revert diff_drive test changes from [pull request 314](https://github.com/ignitionrobotics/ign-gazebo/pull/314)
+    * [pull request 410](https://github.com/ignitionrobotics/ign-gazebo/pull/410)
```

Comment:
This is referring to a commit in #410, which is actually a forward port PR (this commit was changing something for the forward port). Should this be in the changelog, or should I take it out?

Reply:
You can leave this out.

Reply:
Sounds good; removed in https://github.com/ignitionrobotics/ign-gazebo/pull/485/commits/c2237b5c6a12794e5e55fc003a000b7165da2794.